src/components/LogseqCopilot.tsx

```diff
@@ -9,6 +9,15 @@ const LogseqCopilot = ({ graph, pages, blocks }) => {
     Browser.runtime.sendMessage({ type: 'open-options' });
   };
 
+  const groupedBlocks = blocks.reduce((groups, item) => {
+    const group = (groups[item.page.name] || []);
+    group.push(item);
+    groups[item.page.name] = group;
+    return groups;
+  }, {});
+
+  console.log({groupedBlocks, blocks})
```

Comment:
Please remove `console.log()`, if you need output data for debugging, please use `console.debug`, or use `console.info` at background.